content/en/docs/setup/independent/setup-ha-etcd-with-kubeadm.md

```diff
@@ -46,7 +46,7 @@ this example.
 
 1. Configure the kubelet to be a service manager for etcd.
 
-    Running etcd is simpler than running kubernetes so you must override the
+    Running etcd is later than running kubeadm so you must override the
```

Comment:
I think there's still a better way to say this.

Maybe "Since etcd was created first, we need to override the service priority [...]"?

Open to feedback from others...

Reply:
> Since etcd was created first, we need to override the service priority
SGTM

cc @chuckha

Reply:
that sounds fine to me!

Reply:
thanks all .updated